parlai/core/gpt2_helper.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python3
```

Comment:
should we consider a `utils` folder (perhaps not even in `core`)? it feels a little weird to me to have `gpt2_helper` in core

Reply:
I agree. We'll move things in another PR though.